Musakai.Blazor/Dockerfile.Blazor

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+WORKDIR /app
+EXPOSE 80
```

Comment:
I think it might be worth exposing 443 as well if you plan on using HTTPS (unless you will have some proxy in front of the app)

Reply:
Resolved